src/components/table/AposTableRow.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <tr class="apos-table__row" :class="{'is-selected': false }">
+    <td class="apos-table__cell">
+      <AposCheckboxInput :field="field" 
```

Comment:
Since we don't have a written label on this, I think we need to do one of two things:
1. Pass in the row title/label and use it for an `aria-label` attribute on the checkbox ("Toggle [row name]")
2. Separate the actual checkbox input into it's own mini-component, then use that both here and in `AposCheckboxInput`, then _here_ set `aria-labelledby` to the row name .
3. Do option 2, but without creating the mini-component. Just the `aria-labelledby` bit.

Reply:
2 might be ideal, but 1 seems solid.

Reply:
I don't think i want to start breaking down inputs and cherry picking pieces this early .. i think that could lead to inconsistent practices throughout the UI .. going to implement 1